---
float64 dist
float64 velocity_mean
